cmd/entrypoint/subcommands/cp.go

```diff
@@ -0,0 +1,30 @@
+package subcommands
```

Comment:
Missing copyright license headers 😜

Reply:
👍 thanks @pritidesai ; added the license headers.